app/adaptive_tessellation/AdaptiveTessellation.cpp

```diff
@@ -104,6 +105,14 @@ void AdaptiveTessellation::mesh_preprocessing(
         position_image_path,
         normal_image_path,
         height_image_path);
+
+    m_quadric_integral =
+        wmtk::QuadricIntegral(displaced, wmtk::QuadricIntegral::QuadricType::Plane);
```

Comment:
Make sure to default to `Triangle` quadric, and also add an option to switch to `Point` quadrics.